timely_beliefs/beliefs/classes.py

```diff
@@ -312,6 +322,7 @@ def search_session(  # noqa: C901  # todo: remove after removing deprecated argu
         :param beliefs_after: only return beliefs formed after this datetime (inclusive)
         :param beliefs_before: only return beliefs formed before this datetime (inclusive)
         :param source: only return beliefs formed by the given source or list of sources
+        :param most_recent_only: only return the most recent beliefs from each source about each event
```

Comment:
Let's add that this means we'll find the belief with the smallest belief horizon. To be explicit and so it's faster grasped.